web/src/message_list_view.js

```diff
@@ -65,7 +65,7 @@ function same_recipient(a, b) {
     return util.same_recipient(a.msg, b.msg);
 }
 
-function analyze_edit_history(message, last_edit_timestr) {
+function analyze_edit_history(message, last_edit_timestamp) {
```

Comment:
Please separate any renames into it's own commit. It helps in reducing distractions while reviewing.

Reply:
@sayamsamal I am not sure if it is renaming since I am passing different parameters. Earlier it was a string we got from `_get_msg_timestring` and now I am passing the `timestamp` held by `message` object. Should I separate it to another commit?

Reply:
Oh, I see, in that case you can keep the commit as is. No worries.